### DISCUSSION TITLE
Possible to build pointy grid surrounded by a pointy hexagon via `Reactor` meshing?

### DISCUSSION THREAD

**Discussion by veeshy** [idaholab/moose#20338] [Q&A General] [answered]
I want the outer hexagon to have sides parallel to the inner hexagons, is that possible now or should I open a feature request?

<img width="642" alt="image" src="https://user-images.githubusercontent.com/5826161/154592446-b35f9385-488b-4ea0-8c90-f3fecf297921.png">

```
[Mesh]
  [foo]
    type = HexagonConcentricCircleAdaptiveBoundaryMeshGenerator
    num_sectors_per_side = '4 4 4 4 4 4'
    background_intervals = 2
    hexagon_size = 2
    is_control_drum = false
    ring_radii = '1'
    ring_intervals = '3'
  []

  [layout]
    type = PatternedHexMeshGenerator
    inputs = 'foo'
    pattern = '   0 0;
                 0 0 0;
                  0 0'
    hexagon_size = 8
  []
[]

[Problem]
  solve = false
[]

[Outputs]
  exodus = true
[]
```

**Comment by GiudGiud** [comment]
Hello

Can you please try the `pattern_boundary=hexagon` option?
It s documented here:
https://mooseframework.inl.gov/source/meshgenerators/PatternedHexMeshGenerator.html

Guillaume

**Reply by veeshy** [reply]
That’s the default so that’s what’s shown already. I also tried the rotate input which rotates the whole mesh and not just the background hex (granted the doc does say it’s for the whole mesh)

**Reply by GiudGiud** [reply]
Ok i understand better what you want now.
You could try rotating foo, but doubt that s supported.

**Reply by GiudGiud** [reply]
I remember the code to build the outer boundary to be fairly flexible, so if it lets you build the pattern with a rotated unit cell you should be ok for building the boundary

**Comment by veeshy** [comment, marked as answer]
Skimming https://www.osti.gov/biblio/1821454 I don’t think I see any flat hex grid surrounded by a flat hex either
